modules/virtual_network_gateway/variables.tf

```diff
@@ -8,46 +15,16 @@ variable "location" {
   type        = string
```

Comment:
the description says `load balancer`, should be VNG

Reply:
fixed